action.yml

```diff
@@ -74,9 +79,12 @@ runs:
 
         # Make sure we aren't creating a package with zero contents
         $assetCount = (Get-ChildItem -Path ./packaging/content | Measure-Object).Count
-        if ($assetCount -eq 0) {
+        if ( ($assetCount -eq 0) -and (!$env.META)) {
           throw "No assets were found for the deployment package"
         }
+        if ( ($assetCount -gt 0) -and ($env.META)) {
+          throw "Assets were found on meta only package"
+        }
```

Comment:
I'm not sure about this bit. Let's say the connector gets an .exe asset in 6 months. Do you want to find out about it during the release process after you've already tagged?

Reply:
I had the same thought in that it should likely be intentional to be it a meta only package. However, yes, it could change and very likely people will forget to update this. I'll revert this change.